petrel/setup.py

```diff
@@ -13,12 +13,12 @@
 
 PACKAGE = "petrel"
 
-PETREL_VERSION = '0.3'
+PETREL_VERSION = '0.10.0'
 
 def get_storm_version():
     version = subprocess.check_output(['storm', 'version']).strip()
-    m = re.search(r'^(\d\.\d\.\d)(-(\w+))?', version)
-    return m.group(0), m.group(1)
+    m = re.search('^(Storm )?(\d+\.\d+\.\d+)', version)
```

Comment:
This looks like it will break compatibility with some older versions of Storm by removing the 

```
(-(\w+))?
```